src/components/ChatInput/ChatInput.tsx

```diff
@@ -61,7 +62,8 @@ const Icon: StyledComponent<
   background: none;
   border: none;
   border-radius: 50%;
-  right: 0;
+  right: 7%;
```

Comment:
Hello @Lavolp have you fixed the responsiveness issue with % position?

Reply:
I have fixed the position/size issues and amended your commit. I will merge it manually (signed commits issue).